tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -45,41 +43,53 @@ public function test_add_post_syncs_post_data() {
 
 	public function test_add_post_syncs_request_is_auto_save() {
 		//Sync from setup should not be auto save
-		$event = $this->server_event_storage->get_most_recent_event( 'wp_insert_post' );
+		$event = $this->server_event_storage->get_most_recent_event( 'jetpack_save_post' );
 		$this->assertFalse( $event->args[3] );
-		
+
 		Jetpack_Constants::set_constant( 'DOING_AUTOSAVE', true );//define( 'DOING_AUTOSAVE', true );
```

Comment:
Can you please remove this stray comment while you're in here?

Reply:
Will do